src/patternfly/components/Pagination/pagination-options-menu.hbs

```diff
@@ -1,5 +1,5 @@
 {{#> options-menu}}
```

Comment:
Looks like `pf-m-top` needs to be applied to `options-menu` instead of `options-menu-toggle`

Reply:
doh!